get-resource.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash -xe
 #CACHEURL=http://172.22.0.1/images
 
+# Check and set http(s)_proxy. Required for cURL to use a proxy
+if [ ! -z "HTTP_PROXY" ]; then
```

Comment:
Is there any reason to check these first? I wonder if doing something like this would be simpler/clearer?

```
export http_proxy=${http_proxy:-$HTTP_PROXY}
```

What's the effect of setting the variables to an empty string? Does that confuse curl?

Reply:
There isn't any side effect of having an empty string. The single line method is better. Will change it.